src/conponents/Search.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import "../styles/search.css";
+import getImages from "../requests/getImages";
+
+const Search = ({ setSearchResults }) => {
+    const [value, setValue] = useState();
+
+    const handleSubmit = async (event) => {
+        event.preventDefault();
+        setSearchResults(await getImages(value));
+    };
+    
+    return (
+        <>
+    <form className="search-form" onSubmit={handleSubmit}>
+            <input 
+            className="search-input" 
+            type="text" 
+            onChange={(event) => setValue(event.target.value)}
+        />
+        <button className="search-btn" type="submit">
+            Go!
+        </button>
+    </form>
```

Comment:
there are a lot of linting/formatting/indenting problems throughout this project. They could be easily fixed by setting up prettier. You can read how to do that [here](https://blog.bitsrc.io/add-prettier-to-your-project-d7e91ac03d05)